content/user/payment/paypal_troubleshooting.md

```diff
@@ -32,7 +32,7 @@ These are the common configuration errors causing IPN processing to fail:
 5.  Use your browser and go to the Login page of your store. If the page is SSL, do you get any certificate errors in your browser? Test from a couple different computers that you don't normally use. An invalid SSL certificate or one that has errors of any sort, could prevent PayPal from successfully posting the notices to your site.
 6.  Try accessing `https://YOURSITE.com/ipn_main_handler.php` with your browser. If you see PHP errors, those will need to be resolved. If you get a white screen, then the first phase of PayPal™ contacting your site isn't encountering errors. This doesn't mean there aren't any, it just means the initial most obvious steps are working.
 7.  There are [two communication testing tools available in the support forum](http://www.zen-cart.com/forum/showthread.php?t=65680). If you are asking for troubleshooting help, please supply the URL to each of these tools after you have installed them on your site, so we can assess the responses they reveal.
```

Comment:
We should probably update all the `http:` links to `zen-cart.com` to `https`

Reply:
Preferably as a separate PR

Reply:
done

Reply:
oops! sorry.